core/ide/src/main/java/org/alice/stageide/sceneeditor/StorytellingSceneEditor.java

```diff
@@ -463,6 +475,44 @@ public void setSelectedExpression(Expression expression) {
     }
   }
 
+  private void updateOrthographicCameraView(EntityImp targetImplementation) {
+    double targetHeight = targetImplementation.getAxisAlignedMinimumBoundingBox().getHeight();
+    AffineMatrix4x4 targetTransform = targetImplementation.getAbsoluteTransformation();
+    ClippedZPlane picturePlane = new ClippedZPlane();
+
+    AffineMatrix4x4 topTransform = AffineMatrix4x4.createIdentity();
+    topTransform.translation.x = targetTransform.translation.x;
+    topTransform.translation.y = 10;
+    topTransform.translation.z = targetTransform.translation.z;
+    topTransform.orientation.up.set(0, 0, 1);
+    topTransform.orientation.right.set(-1, 0, 0);
+    topTransform.orientation.backward.set(0, 1, 0);
+    this.topOrthoMarkerImp.setLocalTransformation(topTransform);
+    picturePlane.setCenter(0, 0);
+    picturePlane.setHeight(16);
+    this.topOrthoMarkerImp.setPicturePlane(picturePlane);
+
+    AffineMatrix4x4 sideTransform = AffineMatrix4x4.createIdentity();
+    sideTransform.translation.x = targetTransform.translation.x + 10.0f;
+    sideTransform.translation.y = targetHeight / 2;
+    sideTransform.translation.z = targetTransform.translation.z;
+    sideTransform.orientation.setValue(new ForwardAndUpGuide(Vector3.accessNegativeXAxis(), Vector3.accessPositiveYAxis()));
+    this.sideOrthoMarkerImp.setLocalTransformation(sideTransform);
+    picturePlane.setHeight(6);
```

Comment:
For side and front you have overridden the original picture plane height.
If you change the values in the initialization you can drop these as repeats and have no picturePlane logic in this method.